Compilation error with recent Boost versions

```bash
osrm-backend/include/util/static_rtree.hpp: In member function ‘void osrm::util::StaticRTree<EdgeDataT, CoordinateListT, UseSharedMemory, BRANCHING_FACTOR, LEAF_PAGE_SIZE>::MapLeafNodesFile(const boost::filesystem::path&)’:
osrm-backend/include/util/static_rtree.hpp:390:29: error: ‘str’ is not a member of ‘boost’
             throw exception(boost::str(boost::format("Leaf file %1% mapping failed: %2%") %
                             ^
osrm-backend/include/util/static_rtree.hpp:390:40: error: ‘format’ is not a member of ‘boost’
             throw exception(boost::str(boost::format("Leaf file %1% mapping failed: %2%") %
                                        ^
```

--- a/include/util/static_rtree.hpp
+++ b/include/util/static_rtree.hpp
@@ -17,6 +17,7 @@
 #include <boost/assert.hpp>
 #include <boost/filesystem.hpp>
 #include <boost/filesystem/fstream.hpp>
+#include <boost/format.hpp>
 #include <boost/iostreams/device/mapped_file.hpp>
 
 #include <tbb/parallel_for.h>

